interfaces/automated-testing/rid/ingestion.yaml

```diff
@@ -0,0 +1,163 @@
+  title: Remote ID Display Data Ingestion
+  version: 0.0.1
+  description: >-
+    This interface is implemented by every Display Provider wishing to be tested by the automated testing framework.
+    The automated testing suite calls this interface to obtain current remote ID information from the perspective of the Display Provider.
+paths:
+  /display_data:
+    get:
+      parameters:
+        -
+          examples:
+            pyramids:
+              value: '29.97816,31.13296,29.98025,31.13535'
+          name: view
+          description: >-
+            The area of this view: lat1,lng1,lat2,lng2.  The view is the smallest box bounded by
+            the provided corner points.
+          schema:
+            type: string
+          in: query
+          required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetDisplayDataResponse'
+          description: Current remote ID display data was retrieved successfully.
+      operationId: getDisplayData
+      summary: Poll display data
+      description: Request current remote ID display data as would be visualized by a Display Application.
+  /display_data/{id}:
+    get:
+      parameters:
+        -
+          examples:
+            default:
+              value: '1e3adb99-acc9-424f-a04e-a0743538849a'
+          name: id
+          description: >-
+            ID of flight identified in `/display_data`.
+          schema:
+            type: string
+          in: path
+          required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetDetailsResponse'
+          description: Details about the requested flight were retrieved successfully.
+        '404':
+          description: Requested flight was not found.
+      operationId: getDetails
+      summary: Get flight details
+      description: Get the details of a flight identified in `/display_data`.
+components:
+  schemas:
+    GetDisplayDataResponse:
+      description: Response to a request for current data that would be visualized by a Display Application.
+      type: object
+      properties:
+        flights:
+          description: Current information for set of discovered flights whose precise locations are known.
+          type: array
+          items:
+            $ref: '#/components/schemas/Flight'
+          default: []
+        clusters:
+          description: >-
+            Current information for sets of discovered flights whose precise locations are not
+            known.
+          type: array
+          items:
+            $ref: '#/components/schemas/Cluster'
+          default: []
+    GetDetailsResponse:
+      description: Response to a request to get details about a flight.
+      type: object
+    Flight:
+      description: ''
+      required:
+        - id
+        - most_recent_position
```

Comment:
Serial number, registration number, and operation description are all fields in a /flight/details call, not the analog to a /flights query -- this object (`Flight`, a field in `GetDisplayDataResponse`) is the analog to a /flights query.  `GetDetailsResponse` is the response from a query similar to /flight/details and it currently has no fields because none of the automated tests identified in the SUSI document check this data.  Because the data format at the point it is ingested into the automated testing system (roughly the point of the Display Application) is not standardized, we shouldn't assume that all information available at the Display Provider is necessarily available at the Display Application unless that information is strictly required (because a Display Provider may choose not to expose certain information in its Display Application API if it is not required).  In Europe, the serial_number will be required so it may make sense to create an additional test to verify that the Display Application is capable of providing this data to the user, but it should be gated by a flag because not all jurisdictions will require this functionality and it is not required in the standard.  I recommend we implement the core functionality first and then add additional tests after MVP is working, at which point more fields in this API may be warranted.  But, I don't think we need to add them now as fields are easy to add, but hard to move, rename or delete.

Reply:
OK Understood, no problems.